docs/documentation/server_admin/topics/roles-groups/con-comparing-groups-roles.adoc

```diff
@@ -2,7 +2,7 @@
 
 = Groups compared to roles
 [role="_abstract"]
-Groups and roles have some similarities and differences. In {project_name}, groups are a collection of users to which you apply roles and attributes. Roles define types of users and applications assign permissions and access control to roles.
+Groups and roles have some similarities and differences. In {project_name}, groups are a collection of users to which you apply roles and attributes. Roles define types of users and applications, assign permissions and access control to roles.
```

Comment:
also, is "roles define [...] access control to roles" intentional?

Reply:
I'd think think the comma goes here: 

```suggestion
Groups and roles have some similarities and differences. In {project_name}, groups are a collection of users to which you apply roles and attributes. Roles define types of users, and applications assign permissions and access control to roles.
```